charts/finrem-ccd-definitions/values.preview.template.yaml

```diff
@@ -208,7 +212,7 @@ ccd:
       imagePullPolicy: Always
       image: hmctspublic.azurecr.io/ccd/definition-store-api:latest
       environment:
-        DEFINITION_STORE_DB_HOST: ${SERVICE_NAME}-postgresql
+        DEFINITION_STORE_DB_HOST: '{{ tpl .Values.global.postgresHostname $}}'
```

Comment:
will this space cause any issue?

Reply:
it shouldn't, it is how sscs did on their change